.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 4
```

Comment:
So wide! 😢  (I favour 2, but not wedded to it).

Reply:
So do I! Unfortunately that would make every save of a file reformat this repository pretty aggressively, so I opted to conform to the pre-existing width rather than enforce changing it...